### ISSUE TITLE
【reproduce the result】evaluation results are not as good as those published in this repo

### ISSUE THREAD

**Issue by weweus** [skyhehe123/SA-SSD#13]
Hi, is the pretrained model trained on train-val split (3712+3769) ?
same issue as #5 

I notice that contents of the default car_cfg.py  include "kitti_dbinfos_trainval.pkl". And i have trained a model used "kitti_dbinfos_train.pkl", but evaluation results (bev AP) of the model trained on "kitti_dbinfos_trainval.pkl" (3712) are not as good as those published in readme.md of this repo:
```
Car AP@0.70, 0.70, 0.70:
bbox AP:99.37, 93.74, 91.16
bev  AP:96.91, 90.32, 87.75
3d   AP:93.45, 83.94, 81.07
aos  AP:99.31, 93.47, 90.72

Car AP@0.70, 0.50, 0.50:
bbox AP:99.37, 93.74, 91.16
bev  AP:99.38, 95.86, 93.39
3d   AP:99.37, 95.76, 93.30
aos  AP:99.31, 93.47, 90.72
```
By the way, to create "kitti_dbinfos_train.pkl", i modified the create_data.py:
```
create_groundtruth_database(data_path='/home/billy/datasets/KITTI',\
                                db_info_save_path='/home/billy/datasets/KITTI/kitti_dbinfos_train.pkl')
```

**Comment by skyhehe123** [comment]
@weweus The pretrained model is trained on train split only and so as the instance pool for augmentation. The gap between may due to the codebase, in which the optimizer are different. I have just updated the optimizer (Adam with one_cycle scheduler) in the latest commitment. Please check and re-run the script.

**Comment by weweus** [comment]
> @weweus The pretrained model is trained on train split only and so as the instance pool for augmentation. The gap between may due to the codebase, in which the optimizer are different. I have just updated the optimizer (Adam with one_cycle scheduler) in the latest commitment. Please check and re-run the script.

Thx a lot, i will check it later and train with the new config file.

**Comment by GuangshengShi** [comment]
@skyhehe123 I have retrained by lastest released code (One cycle scheduler) with only train set. Still, I found the results are worse than published in Readme.md
Car AP@0.70, 0.70, 0.70:
bbox AP:99.60, 95.93, 93.36
bev  AP:96.79, 90.44, 89.75
3d   AP:93.23, 84.13, 81.25
aos  AP:99.51, 95.63, 92.89
Car AP@0.70, 0.50, 0.50:
bbox AP:99.60, 95.93, 93.36
bev  AP:99.61, 96.10, 93.55
3d   AP:99.59, 96.02, 93.47
aos  AP:99.51, 95.63, 92.89

**Comment by skyhehe123** [comment]
@weweus @GuangshengShi After double checking my published results in this repo, it seems the results are inconsistent with so many others and also deviated from what I claimed in my paper. I finally notice the deviation mainly comes from using the kitti_dbinfos_trainval.pkl when I developing my new codebase and I  rush to put the code online without carefully check. I will update the readme by adapting the results from @Son-Goku-gpu which is more convincing.

**Comment by weweus** [comment]
> @skyhehe123 I have retrained by lastest released code (One cycle scheduler) with only train set. Still, I found the results are worse than published in Readme.md
> Car AP@0.70, 0.70, 0.70:
> bbox AP:99.60, 95.93, 93.36
> bev AP:96.79, 90.44, 89.75
> 3d AP:93.23, 84.13, 81.25
> aos AP:99.51, 95.63, 92.89
> Car AP@0.70, 0.50, 0.50:
> bbox AP:99.60, 95.93, 93.36
> bev AP:99.61, 96.10, 93.55
> 3d AP:99.59, 96.02, 93.47
> aos AP:99.51, 95.63, 92.89

Maybe, you could try to find the best model in all trained models. I found that the evaluation results of Epoch75.pth are better than yours (bev AP[medium]=92.14 under iou=0.7).

**Comment by skyhehe123** [comment]
@weweus 这个就太神奇了。。。
